test/cookie.ts

```diff
@@ -1,10 +1,15 @@
-import test from 'ava';
+import test, {ExecutionContext} from 'ava';
+// @ts-ignore
 import PNG from 'png-js';
 import pify from 'pify';
 import Pageres from '../source';
 import {createCookieServer} from './_server';
 
-async function cookieTest(input, t): Promise<void> {
+interface Cookie {
+	[key: string]: string;
+}
+
+async function cookieTest(input: string | Cookie, t: ExecutionContext): Promise<void> {
```

Comment:
The appropriate input type would be  `string | SetCookie`. I hacked it this way to avoid importing `SetCookie` from `puppeteer`.